packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.scss

```diff
@@ -6,6 +6,10 @@
 .tableContainer {
   padding: 0;
 
+  :global(.cds--data-table) td {
+    text-align: start;
```

Comment:
Please don't use global, make it specific to the data table.

If you want to add global overrides, you can add them in esm-styleguide.
Thanks!

Reply:
@vasharma05 
Done. Suggested changes were applied [here](https://github.com/openmrs/openmrs-esm-core/pull/892)